SoftLayer/CLI/licenses/create.py

```diff
@@ -0,0 +1,33 @@
+"""Order/create a vwmare licenses."""
+# :licenses: MIT, see LICENSE for more details.
+
+import click
+from SoftLayer.managers import ordering
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command()
+@click.option('--key', '-k', required=True, prompt=True, help="The License Key for this specific Account License.")
```

Comment:
Include how to get this `key` value in the help message, and an example of what the key looks like.